Cesium3DTilesContent/src/SubtreeAvailability.cpp

```diff
@@ -528,6 +292,46 @@ bool SubtreeAvailability::isContentAvailable(
       this->_contentAvailability[contentId]);
 }
 
+void SubtreeAvailability::setContentAvailable(
+    const CesiumGeometry::QuadtreeTileID& subtreeID,
+    const CesiumGeometry::QuadtreeTileID& tileID,
+    uint64_t contentId,
+    bool isAvailable) noexcept {
+  uint64_t relativeTileMortonIdx =
+      ImplicitTilingUtilities::computeRelativeMortonIndex(subtreeID, tileID);
+  this->setContentAvailable(
+      tileID.level - subtreeID.level,
+      relativeTileMortonIdx,
+      contentId,
+      isAvailable);
+}
+
+void SubtreeAvailability::setContentAvailable(
+    const CesiumGeometry::OctreeTileID& subtreeID,
+    const CesiumGeometry::OctreeTileID& tileID,
+    uint64_t contentId,
+    bool isAvailable) noexcept {
+  uint64_t relativeTileMortonIdx =
+      ImplicitTilingUtilities::computeRelativeMortonIndex(subtreeID, tileID);
+  this->setContentAvailable(
+      tileID.level - subtreeID.level,
+      relativeTileMortonIdx,
+      contentId,
+      isAvailable);
+}
+
+void SubtreeAvailability::setContentAvailable(
+    uint32_t relativeTileLevel,
+    uint64_t relativeTileMortonId,
+    uint64_t contentId,
+    bool isAvailable) noexcept {
+  this->setAvailable(
+      relativeTileLevel,
+      relativeTileMortonId,
+      this->_contentAvailability[contentId],
```

Comment:
Is it intentional / okay that there's no out-of-bounds check for `contentId`?

Reply:
No, that's not good. I added a bounds check.